setup.py

```diff
@@ -28,6 +28,9 @@ def get_version():
         'scikit-learn>=0.19.1',
         'lalsuite>=6.70',
         'gwsurrogate',
+        'qnm',
+        'sxs',
+        'scri'
```

Comment:
The sxs and scri packages aren't actually required, are they? We just use them in example notebooks, which is why they're installed in the test workflow (which uses nbconvert and runs the nbs to check they work)

Reply:
yeah, just tests and notebooks. I've removed sxs, scri, and qnm from the list of installation requirements